packages/blitz-rpc/src/data-client/rpc.ts

```diff
@@ -93,9 +93,6 @@ export function __internal_buildRpcClient({
       serialized = serialize(params)
     }
 
-    // Create a new AbortController instance for this request
-    const controller = new AbortController()
-
```

Comment:
Hmm, shouldn't we still use this in case `signal` argument is undefined?

Reply:
Unless we want to wire it up to something, this won't do anything by itself so 🤷‍♂️